urbanairship-react-native/js/UAEventEmitter.ts

```diff
@@ -1,33 +1,38 @@
 /* Copyright Airship and Contributors */
 
-// @flow
 'use strict';
 
-import {
-  NativeModules,
-  NativeEventEmitter,
-  Platform
-} from 'react-native';
+import { NativeModules, NativeEventEmitter, EmitterSubscription, Platform } from "react-native";
 
+/**
+ * @hidden
+ */
 const UrbanAirshipModule = NativeModules.UrbanAirshipReactModule;
 
-class UAEventEmitter extends NativeEventEmitter {
-
+/**
+ * Custom native event emitter with additional Android behavior
+ *
+ * @hidden
+ */
+export class UAEventEmitter extends NativeEventEmitter {
   constructor() {
     super(UrbanAirshipModule);
   }
 
-  addListener(eventType: string, listener: Function, context: ?Object): EmitterSubscription {
+  addListener(eventType: string, listener: (...args: any[]) => any, context?: object | null | undefined): EmitterSubscription {
     if (Platform.OS === 'android') {
       UrbanAirshipModule.addAndroidListener(eventType);
     }
     return super.addListener(eventType, listener, context);
   }
 
-  removeAllListeners(eventType: string) {
+  removeAllListeners(eventType?: string) {
```

Comment:
why would event type be nullabale?

Reply:
so that you can call `removeAllListeners()` to remove all listeners for all event types. its typed that way in the base class: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/index.d.ts#L215

Reply:
What about https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/index.d.ts#L9256 ?

Reply:
I am using `"@types/react-native": "^0.62.13"` and get a ts build error if i try to call it with a nullable